### DISCUSSION TITLE
[ Section 3.10 - Primeiro Script] Erro Unsupported argument

### DISCUSSION THREAD

**Discussion by cfilipee** [chgasparoto/curso-aws-com-terraform#115] [Q&A]
### Estou tentanto criar um bucket, porém, quando uso o comando (terraform plan) estou recebendo o seguinte erro:
![erro01](https://user-images.githubusercontent.com/42581869/174465850-1342a2af-42b0-4b4f-8984-0ff2e72947fc.png)

```
provider "aws" {
  region = "us-east-1"
}

resource "aws_s3_bucket_acl" "b" {
  bucket = "meubucketcaiofilipee814365-2"
  acl    = "private"


  tags = {
    Name        = "Mybucket"
    Environment = "Dev"
    Managedby   = "Terraform"
  }
}

```

**Comment by chgasparoto** [comment]
@cfilipee você está tentando criar uma ACL (access control list) de um bucket e não um bucket. Este recurso não aceita tags.

O recurso que você precisa utilizar para criar um bucket, é este [aqui](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket).

```hcl
provider "aws" {
  region = "us-east-1"
}

resource "aws_s3_bucket" "b" {
  bucket = "meubucketcaiofilipee814365-2"
  acl    = "private"


  tags = {
    Name        = "Mybucket"
    Environment = "Dev"
    Managedby   = "Terraform"
  }
}
```
